pkg/controller/pxcbackup/controller.go

```diff
@@ -100,6 +107,12 @@ type ReconcilePerconaXtraDBClusterBackup struct {
 	serverVersion       *version.ServerVersion
 	chLimit             chan struct{}
 	bcpDeleteInProgress *sync.Map
+	log                 logr.Logger
+}
+
+func (r *ReconcilePerconaXtraDBClusterBackup) logger(name, namespace string) logr.Logger {
+	return log.NewDelegatingLogger(r.log).WithValues("controller", "perconaxtradbclusterbackup",
+		"cluster", name, "namespace", namespace)
```

Comment:
in case of PerconaXtraDBClusterBackup reconciler, it is not a name of cluster, but name of backup CR
```suggestion
		"backup", name, "namespace", namespace)
```